### PR TITLE
fix for #4648 tenmanga.com

### LINKED ISSUE #4648: [https://tenmanga.com/] Connector not working
**Did you read [the troubleshooting guide](https://hakuneko.download/docs/troubleshoot/)**
Yes

**Is the website of the connector working properly / are you able to see the manga within your browser**
Yes

**Describe the bug**
HakuNeko is not loading any manga when this connector is chosen.

**To Reproduce**
1. Name of connector
2. Name of manga (if connector update is working)
3. Name of chapter (if manga update is working)
4. Copy any relevant error message that appears in a popup or (advanced) within the console (press F12)

**Screenshots**
If complex to describe, add screenshots to help explain your problem.

**HakuNeko (please complete the following information):**
 - Version 8.3.4

### PR DESCRIPTION
**Pull request by vonProteus**
Resolves #4648

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,8 +31,8 @@
   "scripts": {
     "postinstall": "cd src/app && npm install --no-optional",
     "start": "npm run start:dev",
-    "start:dev": "electron . --update-url=DISABLED --cache-directory=./src/web --disable-http-cache",
-    "start:build": "npm run build:web && electron . --update-url=DISABLED --cache-directory=./build/latest",
+    "start:dev": "electron . --update-url=DISABLED --cache-directory=./src/web --disable-http-cache --remote-debugging-port=9200",
```
Comment (27136773) [inline package.json]: Why setting/changing the remote debugging port?
It seems not related to TenManga bug

Comment (vonProteus) [reply package.json]: This is the only way I was able to connect the deuger in webstorm
But this is my first contact with an electron
Should I roll back this change?

Comment (27136773) [reply package.json]: Yes, leave the remote debugging off (it could have side-effects and a website may be able to detect the browser is in automation mode => bot detection).

You can use the chrome debugger by pressing F12 in HakuNeko, or if really want remote debugging, use the command `npm run start:dev -- --remote-debugging-port=9200` to start HakuNeko from your IDE.

**Review by 27136773 — APPROVED** [review]
